amqp.go

```diff
@@ -481,6 +483,14 @@ func (c *Client) initconsumer(cs *consumer) error {
 	if err != nil {
 		return err
 	}
+
+	if cs.prefetchCount > 0 {
+		err = channel.Qos(
+			cs.prefetchCount, // prefetchCount
+			0,                // prefetchSize
+			false,            // global: false => count applied separately to each new consumer on the channel
+		)
```

Comment:
I'm puzzled by the fact that this function gets a (possibly nil) error back from several function calls, but doesn't do anything with the error code making a subsequent function call and possibly overwriting whatever error information was present before. I would at least expect it to log the error. Since the function doesn't currently do anything with the errors, a change probably isn't required now. I'd kind of like to see this updated at some point in the future, though.

Reply:
I agree, it seems like these errors should probably be logged, but I wouldn't want to add that logging without some testing, just in case the error messages get out of control.

So I'll skip it for now in this PR.